clang/test/CodeGen/aarch64-sme2-intrinsics/acle_sme2_ldr_str_zt.c

```diff
@@ -0,0 +1,51 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py
+
+// REQUIRES: aarch64-registered-target
+
+// RUN: %clang_cc1 -triple aarch64-none-linux-gnu -target-feature +sme2 -S -disable-O0-optnone -Werror -Wall -emit-llvm -o - %s | opt -S -p mem2reg,instcombine,tailcallelim | FileCheck %s
```

Comment:
I think in this case we can kill off the RUN lines for the overloaded forms because in the ACLE they are never overloaded.

Reply:
Done.